src/natcap/invest/coastal_vulnerability.py

```diff
@@ -2760,6 +2701,47 @@ def polygon_to_lines(geometry):
     return line_list
 
 
+def _ogr_to_geometry_list(vector_path):
+    """Convert an OGR type with one layer to a list of shapely geometry.
+
+    Iterates through the features in the `vector_path`'s first layer and

```

Comment:
This is also pretty trivial, but would it be possible to use RST's double-backticks for inline code rather than mardown's single-backticks?